war/src/main/webapp/css/style.css

```diff
@@ -1295,25 +1295,25 @@ TABLE.fingerprint-in-build TD {
   border: 1px solid #5788a1;
 }
 
-#plugins tr.has-dependants input.uninstall {
+#plugins tr.has-dependents input.uninstall {
```

Comment:
This change (this css class) technically could break anyone who's styling based on the ID. I don't think that's a serious concern.

Reply:
I agree, should be really limited as would require to have something like

```html
<table id="plugins">
  <tr class="has-dependants"/>
</table>
```

which seems pretty specific.